library/src/main/java/com/google/android/material/motion/runtime/Scheduler.java

```diff
@@ -156,6 +160,18 @@ public void commitTransaction(Transaction transaction) {
     }
   }
 
+  /**
+   * Adds a plan to this scheduler. Uses {@link PlanInfo} to join the plan and the target together.
```

Comment:
I would remove internal details "Uses {@link PlanInfo} to join the plan and the target together" from the public documentation.